### ISSUE TITLE
Should we combine the meme commands?

### ISSUE THREAD

**Issue by TristanWiley** [TristanWiley/So-Chat-Commander#50]
Tableflip, kiddo, shruggie. Should they be combined into one command?

Thoughts?

**Comment by drobilc** [comment]
It is not worth the time if we're only going to have 3 emojis...

**Comment by TristanWiley** [comment]
Right, but at what point should combine them? If ever?On Jan 24, 2017 13:19, drobilc <notifications@github.com> wrote:It is not worth the time if we're only going to have 3 commands...

**Comment by drobilc** [comment]
5, 10? I don't know... If we're gonna be adding new emojis it would probably be better to create an emoji popup selector (like quick chat in Call of Duty).

**Comment by TristanWiley** [comment]
True, very true. I don't have the time to do that unfortunately.
